packages/terra-navigation-side-menu/CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ Unreleased
 ### Changed
 * Wdio tests changed to use themeCombinationOfCustomProperties
 
+* Add optional toolbar prop that displays below the side menu action header
```

Comment:
Terra follows [these format guidelines](https://keepachangelog.com/en/1.0.0/) for writing change log entries. Lets add the heading `### Added` for this entry.